harvester_e2e_tests/integrations/test_4_vm_backup_restore.py

```diff
@@ -605,9 +605,227 @@ def test_restore_replace_with_vm_shutdown_command(
         spec = api_client.backups.RestoreSpec.for_existing(delete_volumes=True)
         code, data = api_client.backups.restore(unique_vm_name, spec)
         assert 201 == code, f'Failed to restore backup with current VM replaced, {data}'
-        vm_getable, (code, data) = vm_checker.wait_getable(unique_vm_name)
+
```

Comment:
Why remove `wait_getable`?
Won't possible to hit https://github.com/harvester/tests/pull/1419?

Reply:
Thanks for the check. 
I think that would probably because during the rebase process I miss something for not adding this check back. 
Just added the `wait_getable` action back to the original places.